xproc/src/main/xml/specification.xml

```diff
@@ -404,7 +404,8 @@ XProc pipeline as a map (<code>map(xs:QName, item()*)</code>).</termdef></para>
               properties for the document. If present, it's value <rfc2119>must</rfc2119> be of type
                 <code>map(xs:QName, item()*)</code>. <error code="D0070">It is a <glossterm>dynamic
                   error</glossterm> if a value is assigned to the <code>serialization</code>
-                document property that is not of type <code>map(xs:QName, item()*)</code>.</error>
+                document property can not be converted into <code>map(xs:QName, item()*)</code> according
```

Comment:
… document property can not be converted …
→
… document property that cannot be converted …

Otherwise approve